planning/behavior_path_planner/CMakeLists.txt

```diff
@@ -17,18 +17,18 @@ ament_auto_add_library(behavior_path_planner_node SHARED
   src/turn_signal_decider.cpp
   src/scene_module/scene_module_bt_node_interface.cpp
   src/scene_module/side_shift/side_shift_module.cpp
-  src/scene_module/side_shift/util.cpp
   src/scene_module/avoidance/avoidance_module.cpp
   src/scene_module/avoidance/avoidance_utils.cpp
   src/scene_module/avoidance/debug.cpp
   src/scene_module/lane_following/lane_following_module.cpp
   src/scene_module/lane_change/external_request_lane_change_module.cpp
   src/scene_module/lane_change/lane_change_module.cpp
-  src/scene_module/lane_change/util.cpp
-  src/scene_module/lane_change/debug.cpp
   src/scene_module/pull_over/pull_over_module.cpp
   src/scene_module/pull_out/pull_out_module.cpp
   src/scene_module/scene_module_visitor.cpp
+  src/util/lane_change/util.cpp
+  src/util/lane_change/debug.cpp
```

Comment:
@satoshi-ota 
I would like to suggest to have a separate folder for debug. 
Placing debug in utils might confuse others since the debug is usually reserved for developer purpose only. 

I'd love to hear your thoughts on the matter

Reply:
@zulfaqar-azmi-t4 Thanks for your comment :+1: 

> I would like to suggest to have a separate folder for debug.
Placing debug in utils might confuse others since the debug is usually reserved for developer purpose only.

Sure. You're right. Do you have any idea about the right place of (avoidance/lane_change) debug.cpp?

Reply:
`src/util/debug/~` ? :thinking:

Reply:
maybe `src/debug/lane_change/debug.cpp` -> but this is a bit confusing

Reply:
or `src/marker_utils/lane_change/debug.cpp`

Reply:
> or src/marker_utils/lane_change/debug.cpp

The latter idea sounds good to me :+1: I fix it.

Reply:
Thank you!

Reply:
@zulfaqar-azmi-t4 move lane change's debug.cpp (+ debug_utilities.cpp) to `marker_uti/~`
Could you check the [commit](https://github.com/autowarefoundation/autoware.universe/pull/2927/commits/848cbb0f6bf12b466c75949f959a1d53ca5ab7d0) ?

Reply:
nice! Thank you so much